#cloud-config
#voir : https://cloudinit.readthedocs.io/en/latest/reference/examples.html
hostname: {{vm_name}}
#fqdn: Node1.kube.test
manage_etc_hosts: true
disable_root: false
users:
  - name: {{user_name}}
    shell: /bin/bash
    lock-passwd: false
    ssh_pwauth: True
    chpasswd: { expire: False }
    sudo: ALL=(ALL) NOPASSWD:ALL
    groups: users, admin
    ssh-authorized-keys:
      - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCePts/+nGMzelBoBVEepowX2yee572lViBOxveg6anMhUQQp08X0cUTeIuJckcrnvrF9p/3eMxSTkB/QdB9sQioNJdMAFMziAjEJwyunwHxz3MLr/t40/O28rYlP3wXemQ1keI+OjXSjoMtaNlcGq7pTDXFY/ypLyAwEarIy0mEw6PU9pfzSKDLvK6LYtRC1Y/TeZzC5Y45RVLBu204PgTutx+xJswX9LUhTdoSJt960Im8UHNcnapCy2p05UujiV13I5RsvmVQpU3JXlJRmYfaexyPuI7C0wn0E3hHL9oDEnEsOESVHsi96adRbr1rEJF8xajbuwgDYvHl/5bzpKE5GMhdT4hsxBkIeMIWYMyiA6f+DRRXkB1b86OLByg1pkRj2POpUHNP1WBeHxN+1Z0O9KWfKQ8DK5KpZaRpe+N8z6gAX2+Y5t+WWXzHhQciXNzc4LtPKvz9bc0zJU8L8XPZNq5GLfpS77CABhM1QYwGTbVPCVOXT8rjVHsNS7T5Is= vjo@gaia


chpasswd:
  list: |
     {{user_name}}:{{user_pwd}}
     root:{{user_pwd}}
  expire: False

runcmd:
  - [ cp, /home/{{user_name}}/.ssh/authorized_keys, /root/.ssh/authorized_keys ]

# package_update: true
# packages:
#    - dnf-automatic
#    - fail2ban

#systemctl status fail2ban

# written to /var/log/cloud-init-output.log
final_message: "The system is finally up, after $UPTIME seconds"
